### PR TITLE
create the Tx `TransactionType` 

### PR DESCRIPTION
**Pull request by Pantani**
Issue [292](https://github.com/trustwallet/blockatlas/issues/292)

### CODE REVIEW

File: tx.go [hunk 1]
```diff
@@ -14,6 +14,7 @@ const (
 type Direction string
 type Status string
 type TokenType string
+type TransactionType string
```
Comment (vikmeup) [inline tx.go]: would be good to add transactions types into constant?

File: observer/dispatcher.go [hunk 2]
```diff
@@ -26,7 +26,7 @@ func (d *Dispatcher) Run(events <-chan Event) {
 
 func (d *Dispatcher) dispatch(event Event) {
 	action := DispatchEvent{
-		Action: event.Tx.Type,
+		Action: string(event.Tx.Type),
```
Comment (vikmeup) [inline observer/dispatcher.go]: should action use `TransactionType ` ?

File: tx.go (hunk 1 above)
Comment (Pantani) [reply tx.go]: yes!! done!!

File: observer/dispatcher.go (hunk 2 above)
Comment (Pantani) [reply observer/dispatcher.go]: done!

**Pull request merged**